librz/include/rz_analysis.h

```diff
@@ -737,6 +738,15 @@ typedef struct rz_analysis_var_t {
 	int argnum;
 } RzAnalysisVar;
 
+typedef struct rz_analysis_var_global_t {
+	char *name; // name of the variable
+	ut64 addr; // address of the global variable
+	RzType *type; // type of the variable
+	RzVector /*<RzAnalysisVarAccess>*/ accesses; // ordered by offset, touch this only through API or expect uaf
```

Comment:
Is this actually needed? Normally accesses to a global variable would just be represented by xrefs. Special care must be taken when xrefs do not go exactly to the variable's address but at another one that still intersects that variable's type, but that can be solved too.